lib/library_web/templates/component/book.html.eex

```diff
@@ -1,13 +1,20 @@
 <div class="w-100 db v-top mb2 pa3 bs-bb bg-white link dark-grey flex">
-  <%= img_tag(assigns.book.thumbnail, alt: "#{assigns.book.title} cover image", class: "w3-5 pt1 h-100 shadow-4 mr3 dib") %>
+  <div class="w-100 w-image mr3">
+    <%= img_tag(Map.get(assigns.book, :thumbnail_small) || static_path(@conn, "/images/null-image.png"), alt: "#{assigns.book.title} cover image", class: "mt1 h-100 h-image shadow-4") %>
```

Comment:
Would be good to move the logic for finding the right image to the view to keep the templates clean.